tests/plugins/apps/browser/test_chrome_plugin.py

```diff
@@ -0,0 +1,34 @@
+from dissect.target.plugins.apps.browser import chrome
+from tests._utils import absolute_path
+
+
+def test_chrome(target_win, fs_win, tmp_path, target_win_users):
+    chrome_db = absolute_path("_data/plugins/apps/browser/chrome/History.sqlite")
```

Comment:
Personally, I don't like the use of fixtures in this way. Yes, it cleans up the test, but it may also be very confusing how the test is constructed (for me, it was). Hence, I would leave the test logic all together.

I you prefer to move this setup logic nonetheless, I think you can do two things:
- create a fixture file that belongs only to the browsers tests, and don't add the fixture in one god file that holds all fixtures. Place this fixture file in the same directory as the tests, and eventually add a prefix to indictate it is a fixture (e.g. `fixture_chrome.py`);
- refactor the tests so we can use the native setUp hook: https://docs.python.org/3/library/unittest.html. I believe this is out of scope for this pr, however.

Reply:
I think your second point is indeed out of scope for this PR as it would require significant refactoring, since you are essentially switching from one testing framework to another.

I do get your first point, so i'd like to propose moving the contents of `__setup()` to a `@pytest.fixture` decorated function within the respective test file itself. This avoids putting more, not globally used fixtures in `conftest.py`, is in line with how we currently handle these specifics, arguably as readable as the current `__setup()` way and keeps the test itself clean. 

To accomplish this, you could use one of the examples above or take some inspiration from other test files containing fixtures.